src/__tests__/useSet.spec.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
This kind of Pull request is unacceptable @iampika . This is a direct rip off from `react-use` with minor rearrangements.